og.og_permissions.yml

```diff
@@ -0,0 +1,14 @@
+update group:
+  title: 'Edit group'
+  description: 'Edit the group. Note: This permission controls only node entity type groups.'
+  'default role':
+    - OG_ADMINISTRATOR_ROLE
```

Comment:
We can address this later on but we should stop using our OG constants in yaml files and other data where it works purely because of environment at the time.